lib/util.js

```diff
@@ -1,8 +1,11 @@
 /*
  * Copyright (c) 2018-2020 Digital Bazaar, Inc. All rights reserved.
  */
-import {asn1, oids, util} from 'node-forge';
-const {ByteBuffer} = util;
+
+const DER_PRIVATE_KEY_PREFIX = Buffer.from(
+  '302e020100300506032b657004220420', 'hex');
+
+const DER_PUBLIC_KEY_PREFIX = Buffer.from('302a300506032b65700321', 'hex');
```

Comment:
We should put the extra zero byte prefix here (note the additional trailing '00'):

```suggestion
const DER_PUBLIC_KEY_PREFIX = Buffer.from('302a300506032b6570032100', 'hex');
```

Reply:
Addressed here: https://github.com/digitalbazaar/ed25519-verification-key-2020/pull/5/commits/c76e618e9d4d8866bd83b1f468a8f4ace5980944